config/locales/translations/zendesk_apps_support.yml

```
@@ -627,3 +627,15 @@ parts:
       key: "txt.apps.admin.error.app_build.password_parameter_type_deprecated"
       title: "App builder job: Password parameter type is deprecated"
       value: "Password parameter type can no longer be used. Use Secure settings instead. Learn more: %{link}."
+  - translation:
+      key: "txt.apps.admin.error.app_build.scope_requires_secure_parameter"
+      title: "App builder job: scopes can be defined only on secure parameters"
+      value: "Scopes can only be defined on secure parameters."
```

How about using `field` like in `txt.apps.admin.error.app_build.unacceptable_boolean` to indicate the invalid parameter?

This applies to the other error messages as well.

```suggestion
      value: "%{field} cannot be defined in non-secure parameter."
```